{{cookiecutter.project_slug}}/_/deployment/python/deployment/kubernetes/database/base/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  component: database
+resources:
+- ../../base
```

Comment:
I understand, this is for the `commonLabels`. But it reads weird. Can we find a more conventional solution?

Or is it good like this and we should find a way to make it more self-explanatory?

Reply:
Sorry, I don't understand. How is this for the `commonLabels`?

I guess it's a copy/paste error and should be removed.

Reply:
There is a `base` folder outside the database and application directory which contains a `kustomization.yaml` with the `app` common label (used for application and database).

I would say it is wrong to deploy infrastructure with the application but since this should be a simple demo, that is the easiest way.

On the other hand I could move the `app` label to application and database `kustomization.yaml`.

Reply:
Ah, I see. I vote for `commonLabels` in each kustomization.yaml.

Reply:
Updated that 👍